docs/source/usage.rst

```diff
@@ -3,108 +3,81 @@
+:ref:`configuration files <config-files>`.
 
+By default, ``repo2docker`` will assume you are using
+Python 3.6 unless you include the version of Python in your
+:ref:`configuration files <config-files>`.  ``repo2docker`` support is best with
+Python 2.7, 3.5, and 3.6.  In the case of this repo, a Python version is not
+specified in their configuation files and Python 3.6 is installed.
 
-The simplest invocation of ``repo2docker`` performs two steps:
+`Python Data Scinece Handbook <https://github.com/jakevdp/PythonDataScienceHandbook>`_
+uses a `requirements.txt file <https://github.com/jakevdp/PythonDataScienceHandbook/blob/master/requirements.txt>`_
+to specify its Python environment. ``repo2docker`` uses ``pip`` to install
+dependencies listed in the ``requirement.txt`` in image. To learn more about
+configuration files in ``repo2docker`` visit :ref:`config-files`.
 
-1. builds a Docker image from a git repo
-2. runs a Jupyter server within the image
+When the image is built, a message will be output to your terminal::
 
-This two step process enables you to build an image and run it so you can
-explore the repository's contents.
+  Copy/paste this URL into your browser when you connect for the first time,
+  to login with a token:
+      http://0.0.0.0:36511/?token=f94f8fabb92e22f5bfab116c382b4707fc2cade56ad1ace0
 
-The **command** used is::
+Pasting the URL into your browser will open Jupyter Notebook with the
+dependencies and contents of the source repository in the built image.
 
-  jupyter-repo2docker <URL-or-path to repo>
+Because JupyterLab is a server extension of the classic Jupyter Notebook server,
+you can launch JupyterLab by opening Jupyter Notebook and visiting the
+```/lab`` to the end of the URL:
 
-where ``<URL-or-path to repo>`` provides a URL or path to the source repository.
+.. code-block:: none
 
-For example, use the following to build an image and launch a Jupyter Notebook
-server::
+http(s)://<server:port>/<lab-location>/lab
 
-  jupyter-repo2docker https://github.com/jakevdp/PythonDataScienceHandbook
+To switch back to the classic notebook, add ``/tree`` to the URL:
 
-When the example completes building (which may take a few minutes), a message will
-be output to your terminal::
+.. code-block:: none
 
-  Copy/paste this URL into your browser when you connect for the first time,
-  to login with a token:
-      http://0.0.0.0:36511/?token=f94f8fabb92e22f5bfab116c382b4707fc2cade56ad1ace0
+http(s)://<server:port>/<lab-location>/tree
 
-If you copy/paste that URL into your browser you will see a Jupyter Notebook with the
-contents of the source repository which you have just built.
+To learn more about URLs in JupyterLab and Jupyter Notebook, visit
+`starting JupyterLab <http://jupyterlab.readthedocs.io/en/latest/getting_started/starting.html>`_.
 
-Using the ``--debug`` and ``--no-build`` parameters
----------------------------------------------------
+``--debug`` and ``--no-build``
```

Comment:
I'd keep some text describing what these `--weird-double-dash-thing-is`. Maybe "Advanced command-line arguments: `--debug` and `--no-build`?